exporters/trace/jaeger/src/main/java/io/opencensus/exporter/trace/jaeger/JaegerExporterHandler.java

```diff
@@ -0,0 +1,273 @@
+  private static final Tracer tracer = Tracing.getTracer();
+
+  private static final Function<? super String, Tag> stringAttributeConverter =
+      new Function<String, Tag>() {
+        @Override
+        public Tag apply(final String value) {
+          final Tag tag = new Tag();
+          tag.setVType(TagType.STRING);
+          tag.setVStr(value);
+          return tag;
+        }
+      };
+
+  private static final Function<? super Boolean, Tag> booleanAttributeConverter =
+      new Function<Boolean, Tag>() {
+        @Override
+        public Tag apply(final Boolean value) {
+          final Tag tag = new Tag();
+          tag.setVType(TagType.BOOL);
+          tag.setVBool(value);
+          return tag;
+        }
+      };
+
+  private static final Function<? super Long, Tag> longAttributeConverter =
+      new Function<Long, Tag>() {
+        @Override
+        public Tag apply(final Long value) {
+          final Tag tag = new Tag();
+          tag.setVType(TagType.LONG);
+          tag.setVLong(value);
+          return tag;
+        }
+      };
+
+  private static final Function<Object, Tag> defaultAttributeConverter =
+      new Function<Object, Tag>() {
+        @Override
+        public Tag apply(final Object value) {
+          final Tag tag = new Tag();
+          tag.setVType(TagType.STRING);
+          tag.setVStr(value.toString());
+          return tag;
+        }
+      };
+
+  // Re-usable buffers to avoid too much memory allocation during conversions.
+  // N.B.: these make instances of this class thread-unsafe, hence the above
+  // @NotThreadSafe annotation.
+  private final byte[] spanIdBuffer = new byte[SpanId.SIZE];
+  private final byte[] traceIdBuffer = new byte[TraceId.SIZE];
+  private final byte[] optionsBuffer = new byte[Integer.SIZE / Byte.SIZE];
+
+  private final HttpSender sender;
+  private final Process process;
+
+  JaegerExporterHandler(final HttpSender sender, final Process process) {
+    this.sender = checkNotNull(sender, "Jaeger sender must NOT be null.");
+    this.process = checkNotNull(process, "Process sending traces must NOT be null.");
+  }
+
+  @Override
+  public void export(final Collection<SpanData> spanDataList) {
+    final List<Span> spans = spanDataToJaegerThriftSpans(spanDataList);
+    logger.log(Level.INFO, "About to export " + spans.size() + " traces to Jaeger");
+    exportToJaeger(spans);
+  }
+
+  private List<Span> spanDataToJaegerThriftSpans(final Collection<SpanData> spanDataList) {
+    final List<Span> spans = Lists.newArrayListWithExpectedSize(spanDataList.size());
+    for (final SpanData spanData : spanDataList) {
+      spans.add(spanDataToJaegerThriftSpan(spanData));
+    }
+    return spans;
+  }
+
+  private void exportToJaeger(final List<Span> spans) {
+    try {
+      sender.send(process, spans);
+    } catch (TException e) {
+      tracer
+          .getCurrentSpan()
+          .setStatus(
```

Comment:
This will set the status of current span, which I think might not be owned by the exporter.
Better start your own span at the beginning of the method and end it before exit.

Reply:
Thanks, this makes sense!
I guess I'll also override the default sampler with a probability sampler (with a low probability) to avoid potential infinite loops, like for other exporters.